README.md

```diff
@@ -60,8 +60,12 @@ A list of of AI coding topics.
 
 ## Products & Startups
 
-- [Copilot](https://github.com/features/copilot) + [Copilot X](https://github.com/features/preview/copilot-x)
+### Test Case Generation
```

Comment:
Prefer not to split the startup into sub categories - as they pivot time to time.

Reply:
Got it. Updated.